docs/change-log/index.md

```diff
@@ -13,16 +14,186 @@
+| Spark runtime `gpus`                              | `with_driver_limits` / `with_executor_limits` |
+| `mount_v3io_legacy` (mount_v3io no longer calls it) | `mount_v3io`                       |
+| `mount_v3io_extended`                            | `mount_v3io`                   |
+| `LegacyArtifact` and all legacy artifact types that inherit from it (`LegacyArtifact`, `LegacyDirArtifact`, `LegacyLinkArtifact`, `LegacyPlotArtifact`, `LegacyChartArtifact`, `LegacyTableArtifact`, `LegacyModelArtifact`, `LegacyDatasetArtifact`, `LegacyPlotlyArtifact`, `LegacyBokehArtifact`) | `Artifact` or other artifact classes that inherit from it |
+| `init_functions` in pipelines                    | Add the function initialization to the pipeline code instead |
+| The entire `mlrun/mlutils` library               | `mlrun.framework`                     |
+| `run_pipeline`                                   | `project.run`                                     |
+
+**REST APIs deprecated and removed from v1.3.0 code**
+
+- `pod_status header` from response to `get_log` REST API
+- `client-spec` from response to health API 
+- `submit_pipeline_legacy` REST API
+- `get_pipeline_legacy` REST API
+- Five runtime legacy REST APIs, such as: `list_runtime_resources_legacy`, `delete_runtime_object_legacy` etc.
+- httpdb runtime-related APIs using the deprecated runtimes REST APIs, for example: `delete_runtime_object`
+
+
+### Closed issues
+
+- Fix: Project page displayed an empty list after an upgrade [View in Git](https://github.com/mlrun/ui/pull/1611).
+- Jobs and Workflows pages now display the tag of the executed job (as defined in the API). [View in Git](https://github.com/mlrun/ui/pull/1632).
+- Users with developer and data permissions can now add members to projects they created. (Previously appeared successful in the UI but users were not added). [View in Git](https://github.com/mlrun/ui/pull/1617).
+- Error on Spark ingestion with offline target without defined path (error: `NoneType` object has no attribute `startswith`). Fix: default path defined. [View in Git](https://github.com/mlrun/mlrun/pull/3118).
+- `add_aggregation` over Spark fails with `AttributeError` for sqr and stdvar. [View in Git](https://github.com/mlrun/mlrun/pull/3131).
+- Fix: UI Projects' metrics show N/A for all projects when ml-pipeline is down. [View in Git](https://github.com/mlrun/ui/pull/1613).
+- `project.list_models()` did not function as expected for tags and labels. The `list_artifacts` method now accept a dictionary, and 
+    docstrings were added for httpdb and for MLRunProject methods: both `list_artifacts` and `list_models`. [View in Git](https://github.com/mlrun/mlrun/pull/2988).
+- Fix: Failed MLRun Nuclio deploy needs better error messages.     
+- Fix: Second call to Slack notifier with same webhook does not add another notifier. Resolved by the future (not yet released) notification mechanism. 
```

Comment:
remove?

Reply:
@liranbg How about: 
Fix: Second call to Slack notifier with same webhook does not add another notifier. Resolved by code updates that are part of ongoing development for the notification mechanism (that will be available in a future release).